oar/cli/cmd_image_consistency_check.py

```
@@ -1,99 +1,114 @@
+        """Trigger image consistency check Prow job."""
         task_status = self.statebox.get_task_status(TASK_IMAGE_CONSISTENCY_CHECK)
-        if task_status in [TASK_STATUS_PASS, TASK_STATUS_INPROGRESS]:
-            logger.info(f"Image consistency check already {task_status}, skipping")
+        if task_status == TASK_STATUS_PASS:
+            logger.info("Image consistency check already passed, skipping")
             return
 
-        if self.jh.is_job_enqueue(JENKINS_JOB_IMAGE_CONSISTENCY_CHECK):
-            logger.warning("There is pending job in the queue, please try again later")
-            return
+        if task_status == TASK_STATUS_INPROGRESS:
```

Ruff (0.15.6)

[error] `TASK_STATUS_INPROGRESS` may be undefined, or defined from star imports

(F405)